### PR TITLE
Clarify meaning for rosie copy

### LINKED ISSUE #2397: rosie go: clarify meaning of copy suite
Apparently some users get confused between the `Checkout Suite` (i.e. create a local working copy) and the `Copy Suite` options. We could make this clearer by changing the menu option to `Copy to New Suite`.

### PR DESCRIPTION
**Pull request by wxtim**
Closes #2397

### CODE REVIEW

File: sphinx/tutorial/rose/rosie.rst [hunk 1]
```diff
@@ -36,7 +36,9 @@ system.
    Rosie suites can be created by the command:
 
 :ref:`command-rosie-create`
-   Create a new suite or copy an existing one.
+   - Create a new suite.
+   - Create a new suite, starting with the content of another suite,
+     command-rosie-create using the alias `rose copy`.
```
Comment (wxtim) [inline sphinx/tutorial/rose/rosie.rst:41]:
```suggestion
`rosie copy` is an alias for `rosie create`
```

File: sphinx/tutorial/rose/rosie.rst [hunk 2]
```diff
@@ -36,7 +36,9 @@ system.
    Rosie suites can be created by the command:
 
 :ref:`command-rosie-create`
-   Create a new suite or copy an existing one.
+   - Create a new suite.
+   - Create a new suite, starting with the content of another suite,
+`rosie copy` is an alias for `rosie create`
```
Comment (wxtim) [inline sphinx/tutorial/rose/rosie.rst:41]:
```suggestion
   - `rosie copy` is an alias for `rosie create`
```

File: sphinx/tutorial/rose/rosie.rst [hunk 3]
```diff
@@ -36,7 +36,9 @@ system.
    Rosie suites can be created by the command:
 
 :ref:`command-rosie-create`
-   Create a new suite or copy an existing one.
+   - Create a new suite.
+   - Create a new suite, starting with the content of another suite,
+   - `rosie copy` is an alias for `rosie create`
```
Comment (wxtim) [inline sphinx/tutorial/rose/rosie.rst:41]:
```suggestion
     `rosie copy` is an alias for `rosie create`
```

File: sphinx/tutorial/rose/rosie.rst [hunk 4]
```diff
@@ -36,7 +36,9 @@ system.
    Rosie suites can be created by the command:
 
 :ref:`command-rosie-create`
-   Create a new suite or copy an existing one.
+   - Create a new suite.
+   - Create a new suite, starting with the content of another suite.
+     (`rosie copy` is an alias for `rosie create`)
```
Comment (datamel) [inline sphinx/tutorial/rose/rosie.rst:41]: How about for absolute clarity
```suggestion
     (`rosie copy` is an alias that extends `rosie create`)
```

**Review by datamel — APPROVED** [review]
Minor suggestion, not a big deal if you prefer the original.

**Pull request merged**